script/entry.js

```diff
@@ -10,6 +10,8 @@ if (!script) {
   process.exit(1);
 }
 
+process.argv.splice(2, 1)
```

Comment:
why splice this? seems like a potential gotcha for anyone reading argv from a script that doesn't know this is happening

Reply:
I assumed that scripts loaded via `entry.js` will never need to know that the 1st "argument" is the script name. I removed this line but it seems like either approach may cause confusion so it might be better to do less "magic" (splicing the `argv`).